src/main/kotlin/io/snyk/plugin/services/SnykTaskQueueService.kt

```diff
@@ -81,23 +81,29 @@ class SnykTaskQueueService(val project: Project) {
     }
 
     fun connectProjectToLanguageServer(project: Project) {
-        synchronized(LanguageServerWrapper) {
```

Comment:
Question: Why did you remove the `synchronized` block here? Is it because you bound the service to a project and so this function won't be running in parallel anymore?

Reply:
It was not needed, didn't bring the wanted effect, and it blocks usage of the class, so I removed it to have one possible deadlock less.